### ISSUE TITLE
Avoid Exception in tx_kesearch_lib::renderPreviewImage

### ISSUE THREAD

**Issue by lochmueller** [teaminmedias-pluswerk/ke_search#143]
The "renderPreviewImage" function could result in a "The image ... could not be found and won't be included in frontend output" exception (\TYPO3\CMS\Core\Resource\Exception\FolderDoesNotExistException).
The "renderPreviewImage" function should be check this exception and return an empty string in case of an error.

Regards,
Tim

**Comment by christianbltr** [comment]
Thanks for the hint.
How can I test this?

I uploaded an image to fileadmin/user_upload an included it in the search so that it's displayed in the frontend.

Then I renamed the folder "user_upload". 
The image is not shown anymore.

Debugging the result of 
`cObjGetSingle`
in
https://github.com/teaminmedias-pluswerk/ke_search/blob/master/Classes/lib/class.tx_kesearch_lib.php#L1291
shows that it returns an empty string but does not throw an exception.
